### PR TITLE
Issue #22 UI/UX improvment

### LINKED ISSUE #22: Potential UI/UX Improvements for Your Website
Hi, There I didn't get any idea from what the site is for but I found some bugs and some UI error so to just inform you:

Specific Observations:

**Text Focus**:The wording on the website appears to be out of focus or lacking clarity.
**Alignment**: The title and body text are not centered or aligned consistently.
**Navigation Flow:** The current navigation system requires users to utilize the "back" button to navigate forward, which may be counterintuitive.
**Hover Effect**: The hover effect on the "back" icon could be enhanced with padding to improve visual appeal.

I am interested in contributing to the improvement of your website. If these observations are documented as new issues or sub-issues within your existing system, I would be happy to assist in making the site more user-friendly and visually appealing.
240423_11h40m40s_screenshot

### PR DESCRIPTION
**Pull request by diip3sh**
This changes resolves #22 by changing by for stable css styling.

### CODE REVIEW

File: src/sass/main.sass [hunk 1]
```diff
@@ -74,7 +74,7 @@ body
             width: 100%
             height: 100%
             border: none
-            padding: 50px
+            padding: 0 10px 0 10px
         footer
             padding: 2%
```
Comment (Bashamega) [inline src/sass/main.sass:79]:
```suggestion
            
```

**Review by Bashamega — CHANGES_REQUESTED** [review]

File: src/sass/main.sass (hunk 1 above)
Comment (diip3sh) [reply src/sass/main.sass:79]: If only adding padding without specifying it again have same alignment issue if you want I can change 10px to 2% for left and right

Comment (Bashamega) [reply src/sass/main.sass:79]: No problem, I will merge. Thanks for contributing

Comment (diip3sh) [reply src/sass/main.sass:79]: No problem looking forward to contributing more 🤞🏼

**Pull request merged**